docs/european-how-it-works.md

```diff
@@ -0,0 +1,19 @@
+---
+id: european-how-it-works
+title: European Protocol V1 How it Works 
+sidebar_label: How It Works
+slug: /european-v1/how-it-works
+---
+Unlike American-style options, European options expire at a set time, and cannot be exercised before that time. In exchange for this limitation, our European platform enables two important improvements over its American counterpart:
```

Comment:
it may be worth mentioning at the top that the Euros are under active development